index.html

```diff
@@ -0,0 +1,72 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css" />
+    <title>Portfolio Project</title>
+</head>
+<body>
+
+    <div class="template-1">
+
+        <header>
+            <!-- Toolbar Nav -->
+    <nav class="toolbar-container">
```

Comment:
- [ OTPIOANL ] Your nav tag also includes the title. The nav tag should enclose the navigation menu only. The nav tag is for semantic reasons.

Reply:
Yeah I did Changes too in this case. Thank you for guidance @wayungi